src/components/Nav.jsx

```diff
@@ -24,6 +30,9 @@ export default function Nav() {
         <Link to="/topic/football">
       <button className="topic-button" onClick={handleClick} value="football">
         Football</button></Link>
```

Comment:
These topics are for a previous ticket, but they should not be hard coded like this. These should be retrieved from your backend api and listed dynamically. These are the current topics, but new ones could be added and some could even be removed, so this should not be hard coded values.